### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

File: appengine/cloudsql/package.json [hunk 1]
```diff
@@ -18,36 +18,53 @@
     "lint": "samples lint",
     "pretest": "npm run lint",
     "unit-test": "ava --verbose test/*.test.js",
+    "start-proxy": "cloud_sql_proxy -instances=$INSTANCE_CONNECTION_NAME=tcp:$SQL_PORT &",
```
Comment (jmdobry) [inline appengine/cloudsql/package.json]: Do you need `./cloud_sql_proxy`? Or is it expected that `cloud_sql_proxy` will be on the path somewhere?

Comment (ace-n) [reply appengine/cloudsql/package.json]: The latter.

Comment (jmdobry) [reply appengine/cloudsql/package.json]: Also, shouldn't this script just conditionally start the proxy? What if it's already running?

**Comment by jmdobry** [comment]
LGTM other than last question about conditionally starting the proxy

**Pull request merged**